test/images_test.py

```diff
@@ -48,6 +50,26 @@ def test_imagecube_grad(coords):
     # https://github.com/pytorch/pytorch/issues/50014
     loss.backward()
 
+# test for proper fits scale
+def test_imagecube_tofits(coords):
+    # creating base cube
+    bcube = images.BaseCube(coords=coords)
+
+    # try passing through ImageLayer
+    imagecube = images.ImageCube(coords=coords, passthrough=True)
+
+    # sending the basecube through the imagecube
+    imagecube.forward(bcube.forward())
+
+    # creating output fits file with name 'test_cube_fits_file39.fits'
+    # file will be deleted after testing
+    imagecube.to_FITS(fname='test_cube_fits_file39.fits', overwrite=True)
+
+    # inputting the header from the previously created fits file
+    fits_header = fits.open('test_cube_fits_file39.fits')[0].header
+    os.remove('test_cube_fits_file39.fits')
```

Comment:
For example, if you implement the tmp_path, then this remove line wouldn't be necessary.

Reply:
Just updated this and pushed it to my directory. I think everything looks good now!